### ISSUE TITLE
Batch_norm and Linear layer?

### ISSUE THREAD

**Issue by tosi55** [tiny-dnn/tiny-dnn#959]
Hello, I'm beginner on this project.
I want to execute the batch normalization layer which has the scaling and shifting operation.
With reference to issue #292, I tried to connect the linear_layer after the batch_norm_layer like below, but it seemed different from the original paper's behavior.

    net << conv(32, 32, 5, 1, 6, padding::same)
       << batch_norm(32*32, 6)
       << linear(32*32*6)     // only this number was accepted..
       << relu()   ... 

Will someone tell the correct implementation?
Thank you!
